src/main/java/com/mraof/minestuck/data/loot_table/MSChestLootTables.java

```diff
@@ -29,6 +29,43 @@
 	@Override
 	public void accept(BiConsumer<ResourceLocation, LootTable.Builder> lootProcessor)
 	{
+		/*lootProcessor.accept(LootTables.CHESTS_SIMPLE_DUNGEON, LootTable.builder()
+				.addLootPool(LootPool.builder().name(SUPPLIES_POOL).rolls(RandomValueRange.of(0, 1))
+						.addEntry(LandTableLootEntry.builder(LootTables.CHESTS_SIMPLE_DUNGEON).setPool(SUPPLIES_POOL))
+						.addEntry(ItemLootEntry.builder(MSItems.BLANK_DISK).weight(5).quality(3).acceptFunction(SetCount.builder(RandomValueRange.of(1, 2))))
+						.addEntry(ItemLootEntry.builder(MSItems.HASHMAP_MODUS_CARD).weight(1).quality(1).acceptFunction(SetCount.builder(RandomValueRange.of(0, 1))))
+						.addEntry(ItemLootEntry.builder(MSItems.QUEUE_MODUS_CARD).weight(1).quality(1).acceptFunction(SetCount.builder(RandomValueRange.of(0, 1))))
+						.addEntry(ItemLootEntry.builder(MSItems.QUEUESTACK_MODUS_CARD).weight(1).quality(1).acceptFunction(SetCount.builder(RandomValueRange.of(0, 1))))
+						.addEntry(ItemLootEntry.builder(MSItems.SET_MODUS_CARD).weight(1).quality(1).acceptFunction(SetCount.builder(RandomValueRange.of(0, 1))))
+						.addEntry(ItemLootEntry.builder(MSItems.STACK_MODUS_CARD).weight(1).quality(1).acceptFunction(SetCount.builder(RandomValueRange.of(0, 1))))
+						.addEntry(ItemLootEntry.builder(MSItems.TREE_MODUS_CARD).weight(1).quality(1).acceptFunction(SetCount.builder(RandomValueRange.of(0, 1))))));*/
```

Comment:
Agreed, it will be removed or fixed and un-commented after I figure out how to introduce a new loot pool to a preexisting loot table without overriding the entire loot table

Reply:
The alternative to replacing the loot table with a modified version is to add/replace the generated loot through a global loot modifier (https://mcforge.readthedocs.io/en/1.15.x/items/globallootmodifiers/)

Reply:
A different solution was found and implemented successfully!